notebooks/ODE_solver.ipynb

```diff
@@ -11,13 +11,25 @@
     "import json\n",
```

Comment:
<p>maybe add a new section "Creating extrusion dictionaries for lattice simulations", which would have the 2-state conversion and the WAPL extrusion dictionary croeation, and this  kinetic trajectory WAPL depletion could be moved previous to that section?</p><p> </p><p> </p><h1><br></h1>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Fudenberg-Research-Group/five_state_cohesin/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>